app/livechat/imports/server/rest/inquiries.js

```diff
@@ -0,0 +1,49 @@
+import { Meteor } from 'meteor/meteor';
+
+import { API } from '../../../../api';
+import { hasPermission } from '../../../../authorization';
+import { Users } from '../../../../models';
+import { LivechatInquiry } from '../../../lib/LivechatInquiry';
+
+API.v1.addRoute('livechat/inquiries', { authRequired: true }, {
```

Comment:
I think it would be better renaming this route to `livechat/inquiries.list`, ok?